ReleaseProcessScript/tools/PesterTester/semver_functions.Tests.ps1

```diff
@@ -150,4 +150,29 @@ Describe "semver_functions" {
       Get-Most-Recent-Version $BeforeVersion $RecentVersion | Should Be $RecentVersion
     }
   }
+
+  Context "Is-Semver" {
+    It "Recognizes valid versions correctly" {
+      Is-Semver "1.2.3" | Should Be $TRUE
+      Is-Semver "1.2.3-alpha.23" | Should Be $TRUE
+      Is-Semver "1.2.3-beta.1" | Should Be $TRUE
+      Is-Semver "1.2.3-rc.0" | Should Be $TRUE
+    }
+
+    It "Classifies a version without patch part as invalid" {
+      Is-Semver "v.2" | Should Be $FALSE
+    }
```

Comment:
The title and the body of the test don't necessarily match, I believe this test could be split into 3 different tests to make it even more clear:

v1.2.3 (something with a v prefix) should be false

1 (only major) should be false

1.2 (only major and minor) should be false

Do you agree? Or was this test supposed to test something entirely different that I didn't get?

Otherwise I like how you did this test suite, since it leaves very little to interpretation and tests the common cases quite exhaustively

Reply:
You are correct regarding the first test. This is what happened (and didn't catch my eye when pushing):
* I wanted to test which asserts that `1.2` is false (as you suggested)
* At first, I mistakenly prefixed everything with a `v`
* I deleted the `v` from every test, but for this one I deleted the `1` instead of the `v`

I corrected that mistake and added tests for `1` and `v1.2.3`.